app/static/js/index.js

```diff
@@ -19,13 +19,23 @@ let p = {
 //////////////
 // Controls //
 //////////////
-query.onkeyup = function (e) {
-	if (e.keyCode === 13) {
+p.query.onkeyup = function (e) {
+	if (e.key === "Enter") {
 		e.preventDefault();
 		p.submit.click();
 	}
 };
 
+document.onkeydown = function (e) {
+	// No-op if focus is not on the body (i.e. user is typing in a textbox)
+	if (e.target !== document.body) return;
+
+	if(e.key === "/") {
```

Comment:
Nit: space after if and use single quotes

Reply:
Would it be overkill to set up a linter to catch these things...?

Reply:
Run `./setup_tests.sh` to configure the linter as a commit hook.